packages/twenty-server/src/engine/core-modules/workspace/services/workspace.service.ts

```diff
@@ -152,4 +158,14 @@ export class WorkspaceService extends TypeOrmQueryService<Workspace> {
       );
     }
   }
+
+  private async enableDefaultFeatureFlags(workspaceId: string) {
+    DEFAULT_FEATURE_FLAGS.forEach(async (featureFlagKey) => {
```

Comment:
You can do a batch insert by passing an array instead of calling save n times 👍 
```typescript
    await this.featureFlagRepository.save(
      DEFAULT_FEATURE_FLAGS.map((featureFlagKey) => ({
        key: featureFlagKey,
        value: true,
        workspaceId,
      })),
    );
```